lms/static/js/commerce/credit.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Credit-related utilities
+ */
+var edx = edx || {};
+
+(function ($) {
+    'use strict';
+
+    edx.commerce = edx.commerce || {};
+    edx.commerce.credit = edx.commerce.credit || {};
+
+    edx.commerce.credit.createCreditRequest = function (providerId, courseKey, username, error) {
```

Comment:
@clintonb This worked out really nicely. I like the naming convention, and it looks clean when called from the two locations.

Now it needs Jasmine tests to verify that it does the right thing, for both successful responses and for errors. I thought that you could just adapt the tests from the old receipt_view code, but I see that that has no Jasmine tests either (at least that I could see). There also should be Bok Choy tests that verify the integrated behavior.